index.html

```diff
@@ -26,31 +26,58 @@
       </div>
       <div class="menu">
         <ul>
-          <li> About Me </li>
-          <li> Recent Projects </li>
+          <li class="tab" data-content="projects"><a href="#"> Projects </a></li>
+          <li class="tab" data-content="about"><a href="#"> About Me </a></li>
           <li><a href="https://www.linkedin.com/in/keith-brandtjen-25543b76?trk=nav_responsive_tab_profile" target="_blank" class="icon-linkedin"></a></li>
           <li><a href="https://github.com/KBrandtjen" class="icon-github"></a></li>
         </ul>
       </div>
     </nav>
   </header>
   <main>
-  <section id="projects">
-    <article class="template" data-category="placeholder-category">
+  <section id="projects" class="tab-content">
+    <div>
+      <ul id="filters">
+        <li>
+          <select id="author-filter">
+            <option value="">-- Filter By Author --</option>
+          </select>
+        </li>
+        <li>
+          <select id="category-filter">
+            <option value="">-- Filter by Categories --</option>
+          </select>
+        </li>
+      </ul>
+    </div>
+    <article class="template" data-category="category">
         <img src="">
         <h1>Title</h1>
-        <h2>Author</h2>
+        <h2 class="authorClass">Author</h2>
         <section class="project-description"></section>
           published <time pubdate datetime="2000-01-01">Publish Time</time>
     </article>
   </section>
+  <section id="about" class="tab-content">
+    <h1> Welcome to my blog </h1>
+      <p> My name is Keith Brandtjen. I am a new developer, trying to build my skills and learn as much as I can. Check out my work, let me know what you think. If you want to collaborate, contact me. </p>
```

Comment:
Since the <h1> tag is closed, we can add the <p> tag right below. No need to nest.